hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/replication/ReplicationManager.java

```diff
@@ -878,7 +879,15 @@ protected boolean processContainer(ContainerInfo containerInfo,
       if (!handled) {
```

Comment:
We can set state inside Report about container, initially healthy. And once finished, overwrite the actual state at below logic